finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/Command.scala

```diff
@@ -422,8 +441,8 @@ class CommandCodec extends UnifiedProtocolCodec {
 
   val log = Logger(getClass)
 
-  val encode = new Encoder[Command] {
-    def encode(obj: Command) = Some(obj.toChannelBuffer)
+  val encode = new Encoder[RedisMessage] {
```

Comment:
could you mention in CHANGES that you've added support for pubsub, and also that you're changing these APIs?